test.md

```diff
@@ -267,30 +260,22 @@ This asserts related operation about tables.
 This checks that the last allocated memory has the given size and max value.
 
 ```k
-    syntax Assertion ::= "#assertEmptyMemory"    Int MaxBound String
-                       | "#assertEmptyMemoryAux" Int Int MaxBound String
- // --------------------------------------------------------------------
-    rule <k> #assertEmptyMemory SIZE MAX MSG => #assertEmptyMemoryAux (NEXT -Int 1) SIZE MAX MSG ... </k>
-         <nextMemAddr> NEXT </nextMemAddr>
-
-    rule <k> #assertEmptyMemoryAux ADDR SIZE MAX _ => .  ... </k>
+    syntax Assertion ::= "#assertMemory" Int MaxBound String
+ // --------------------------------------------------------
+    rule <k> #assertMemory SIZE MAX MSG => . ... </k>
          <curModIdx> CUR </curModIdx>
          <moduleInst>
            <modIdx> CUR </modIdx>
-           <memIndices> ( 0 |-> ADDR ) => .Map </memIndices>
+           <memIndices> 0 |-> ADDR </memIndices>
            ...
          </moduleInst>
-         <nextMemAddr> NEXT => NEXT -Int 1 </nextMemAddr>
          <mems>
-           ( <memInst>
-               <mAddr>   ADDR  </mAddr>
-               <mmax>    MAX   </mmax>
-               <msize>   SIZE  </msize>
-               <mdata>   .Map  </mdata>
-               ...
-             </memInst>
-          => .Bag
-           )
+           <memInst>
+             <mAddr>   ADDR  </mAddr>
+             <mmax>    MAX   </mmax>
+             <msize>   SIZE  </msize>
+             ...
```

Comment:
Small regression here: It should still check that `<mdata>` contains exactly `.Map` (i.e., is empty). I had some bugs where memory bytes spilled over, so the idea with this test was to assert that the memory was, in fact, empty (or at least didn't contain any more bytes than the intended ones. So it would be good to have that check.